ui/src/app/components/apps-panel.spec.tsx

```diff
@@ -64,11 +68,14 @@ describe('AppsPanel', () => {
   const appsStub = new AppsApiStub();
   const runtimeStub = new RuntimeApiStub();
   beforeEach(() => {
-    serverConfigStore.set({ config: defaultServerConfig });
-    environment.showAppsPanel = true;
+    serverConfigStore.set({
+      config: { ...defaultServerConfig, enableCromwellGKEApp: true },
+    });
     registerApiClient(AppsApi, appsStub);
+    appsStub.createApp = () => Promise.resolve({});
     registerApiClient(NotebooksApi, new NotebooksApiStub());
     registerApiClient(RuntimeApi, runtimeStub);
+    workspaceStub.billingStatus = BillingStatus.ACTIVE;
```

Comment:
Unfortunately, I think it's still needed because we set `billingStatus` to `INACTIVE` in one of the tests. This causes the underlying `workspaceDataStub`'s `billingStatus` field to become `INACTIVE` for the remainder of the tests. I didn't find any fixes I liked after fiddling with it for a few minutes.

There's a larger problem here that many of our JS tests aren't independent of one another.

If you suggest a good fix for this I'll add it, otherwise I'll leave it as-is.